stdlib/builtins.pyi

```diff
@@ -206,8 +206,12 @@ class int:
     def to_bytes(self, length: SupportsIndex, byteorder: Literal["little", "big"], *, signed: bool = ...) -> bytes: ...
     @classmethod
     def from_bytes(
-        cls, bytes: Iterable[SupportsIndex] | SupportsBytes, byteorder: Literal["little", "big"], *, signed: bool = ...
-    ) -> int: ...  # TODO buffer object argument
+        cls: Type[Self],
+        bytes: Iterable[SupportsIndex] | SupportsBytes,
+        byteorder: Literal["little", "big"],
+        *,
+        signed: bool = ...,
+    ) -> Self: ...  # TODO buffer object argument
```

Comment:
The comment should be moved to the line that defines the `bytes` argument.

Reply:
Done! :)